src/support/execute.ts

```diff
@@ -0,0 +1,26 @@
+import { spawn } from 'child_process'
+
+export async function execute(...args: any[]): Promise<any> {
```

Comment:
Aren't we re-inventing [execa](https://www.npmjs.com/package/execa) here?

Reply:
I took in the other way, I removed `execa` :-)

Reply:
Why?

Reply:
To reduce the number of dependencies installed with the CLI.